From f86a8a0f1a20f6f87cac46f9963a638b96a23a17 Mon Sep 17 00:00:00 2001
From: Rolando Roca <rolandor@valvesoftware.com>
Date: Mon, 10 Feb 2025 09:54:33 -0800
Subject: [PATCH 098/188] Adding new Quectel vid/pid

[from commit a29050b4 by @zhongjun.yu]

(cherry picked from commit a58e3d66757dfec2a2a3826270d6adaa1ad29d87)
---
 drivers/bluetooth/btusb.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/bluetooth/btusb.c b/drivers/bluetooth/btusb.c
index 184e95c16..403cfd435 100644
--- a/drivers/bluetooth/btusb.c
+++ b/drivers/bluetooth/btusb.c
@@ -411,6 +411,8 @@ static const struct usb_device_id quirks_table[] = {
 						     BTUSB_WIDEBAND_SPEECH },
 	{ USB_DEVICE(0x2c7c, 0x0132), .driver_info = BTUSB_QCA_WCN6855 |
 						     BTUSB_WIDEBAND_SPEECH },
+	{ USB_DEVICE(0x2c7c, 0x0137), .driver_info = BTUSB_QCA_WCN6855 |
+						     BTUSB_WIDEBAND_SPEECH },
 
 	/* Broadcom BCM2035 */
 	{ USB_DEVICE(0x0a5c, 0x2009), .driver_info = BTUSB_BCM92035 },
-- 
2.55.0

